src/04-modality-specific-files/01-magnetic-resonance-imaging-data.md

```diff
@@ -217,9 +217,9 @@ sub-<label>/[ses-<label>/]
         sub-<label>[_ses-<label>]_task-<label>[_acq-<label>][_ce-<label>][_dir-<label>][_rec-<label>][_run-<index>][_echo-<index>]_sbref.nii[.gz]
 ```
 
-Imaging data acquired during functional imaging (i.e. imaging which supports
+Imaging data acquired during functional imaging (that is, to imaging which supports
 rapid temporal repetition). This includes but is not limited to task based fMRI
```

Comment:
This is a sentence fragment, so we might as well fix that while we're here. What about:

```suggestion
Functional imaging consists of techniques that support rapid temporal repetition.
This includes but is not limited to task based fMRI
```